README.rst

```diff
@@ -45,6 +45,7 @@ Usage
   -k, --kibana_nested                      Format nested fields in Kibana style.
   -r, --raw_query                          Switch query format in the Query DSL.
   -e, --meta_fields                        Add meta-fields in output.
+  -a, --auth                               Elasticsearch basic authentication in the form of username:password.
```

Comment:
```bash
-u, --url URL                                Elasticsearch host URL. Default is http://localhost:9200.
-a, --auth AUTH                              Elasticsearch basic authentication in the form of username:password.
-i, --index-prefixes INDEX [INDEX ...]       Index name prefix(es). Default is ['logstash-*'].
```
And please fix [header](https://github.com/taraslayshchuk/es2csv/pull/17/files/28735fe508445ac0e69eaa41b02022e04fd32d67..759b59964b5ba4949b0b9c19e5ae2e5f4523ef56#diff-88b99bb28683bd5b7e3a204826ead112R32):
`$ es2csv.py [-h] -q QUERY [-u URL] [-a AUTH] [-i INDEX [INDEX ...]]`